platform/solr/pom.xml

```diff
@@ -23,9 +23,6 @@
     <artifactId>solr</artifactId>
     <packaging>pom</packaging>
     <name>DDF Solr</name>
-    <properties>
-        <saxon.version>9.5.1-5</saxon.version>
```

Comment:
I think that version is being used by Lux inside of Solr.  I think you are right that we were not able to upgrade it due removal of features.  I am not sure we actually use all of those features but it would take some work to remove all of the parts of Lux we do not use to try to get to a newer version of Saxon.  Still might be worth a ticket to investigate again.